### ISSUE TITLE
[Drill]: 'Search quick' ranges checkbox controls present should be accessible via arrow keys of Keyboard.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#86833]
### What happened?

'Search quick' ranges checkbox controls present are not accessible via arrow key instead there are accessible via tab key, because of this user has to press multiple tab key to reach the next control. 

https://github.com/grafana/grafana/assets/93735775/2ccec807-b024-436f-a8dd-21986a0b487d

### What did you expect to happen?

'Search quick' ranges checkbox controls present should be accessible via arrow keys of Keyboard.

### Did this work before?

No

### How do we reproduce it?

1.Open the above URL in edge browser [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1)

2.‘Grafana Play Home’ page will open. 

3.Navigate to ‘Time range’ button & activate it. 

4.Navigate on the time range pop-up by tab key, 

5.Observe the issue here. 

### Environment (with versions)?

Grafana: v11.1.0-69051 (9d44c8e8cf) 
OS: Version 24H2 (OS Build 26100.1) 
Browser: Version 125.0.2506.0 (Official build) dev (64-bit) 

### Grafana platform?

None
